pkgs/development/libraries/libraw/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation {
   pname = "libraw";
-  version = "unstable-2021-12-03";
+  version = "0.20.2.p2";
```

Comment:
There is no such release as 0.20.2.p2. https://github.com/LibRaw/LibRaw/tags

Reply:
Correct.  0.20.2 is the closest release.  "p2" is for adding two patches/commits on the corresponding official branch.

Reply:
Normally this is not done because we don't need to rely on package managers parsing version ranges. Also no one is changing that if a third patch is included and it introduces new versions to repology which don't really exist.

Reply:
I don't really care in this case.  What we package is not 0.20.2 anymore, but it's still pretty close.  Version parsing might be useful for vulnerability scanners, but most likely not in this case.